hydjet2.spec

```diff
@@ -0,0 +1,20 @@
+### RPM external hydjet2 2.x.y 
```

Comment:
please provide the exact version

Reply:
Can we put here the link to the actual version instead of hardcoded number?

Reply:
what do you mean ? can you please give an example

Reply:
Something like
### RPM external hydjet2 version
where the variable version would be determined from the link to the json file on the author's site, or just from the some text file, placed on the author's website.
Mayby ### RPM external hydjet2 http://..../version
Would it work?

Reply:
no, it will not work. One needs to explicitly provide the version to be used here and then you can use the version to auto generate the source url e.g.
```
### RPM externals hydjet2 2.4.3
Source: http://cern.ch/lokhtin/hydjet++/%{n}-%{realversion}.tar.gz
```
So in future when you update the version to e.g. `2.4.4` then package system will try to download  http://cern.ch/lokhtin/hydjet++/hydjet2-2.4.4.tar.gz

Reply:
But only after PR, right? I'm looking a way to avoid of necessity to make PR for every minor update of HYDJET++

Reply:
Ok, I see, thanks!